libs/libtalloc/libtalloc-cache.txt

```diff
@@ -0,0 +1,39 @@
+Checking simple C program: OK
```

Comment:
I though it was an accident ...
I'm not sure this answer file is true for all arch in OpenWrt/LEDE
(at least kernel version will change)

found a package of samba4 (also use waf) and it doesn't use this answer file
https://github.com/tobiaswaldvogel/openwrt-addpack/blob/master/samba4/Makefile
https://forum.openwrt.org/viewtopic.php?id=54490
instead it uses a long list of CONFIGURE_VARS (same thing but we keep everything in the Makefile)